.github/workflows/sca_scan.yaml

```diff
@@ -0,0 +1,32 @@
+name: Snyk Software Composition Analysis Scan
+# This git workflow leverages Snyk actions to perform a Software Composition 
+# Analysis scan on our Opensource libraries upon Pull Requests to Main & 
+# Develop branches.  We use this as a control to prevent vulnerable packages 
+# from being introduced into the codebase. 
+on:
+  pull_request:
+    branches: 
+      - master
+      - develop
+jobs:
+  Snyk_SCA_Scan:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.x]
```

Comment:
Any reason we need to use node 14 here? Might be good to use node 16 to be in parity with the rest of the repo, or even better `16.14.2`.
```suggestion
        node-version: [16.x]
```